### PR TITLE
Implement changes in the GRC based on the decrees from 2 Feb 2021

### PR DESCRIPTION
**Pull request by tukusejssirs**
- Change `martha_of_bethany` to `martha_of_bethany_mary_of_bethany_and_lazarus_of_bethany` (see [decree prot. N. 35/21](https://www.vatican.va/roman_curia/congregations/ccdds/documents/rc_con_ccdds_doc_20210126_decreto-santi_la.html));
- Add the following three celebrations to the GRC (see [decree prot. N. 40/21](https://www.vatican.va/roman_curia/congregations/ccdds/documents/rc_con_ccdds_doc_20210125_decreto-dottori_la.html)):
	- `gregory_of_narek_abbot`;
	- `hildegard_of_bingen_abbess`;
	- `john_of_avila_priest`;
- Add localised celebration name of `gregory_of_narek_abbot` to all supported locales;
- `sk`: Fix localisation of `hildegard_of_bingen_abbess`;
- `pt-br`: Add localised keys for all four celebrations added to the GRC.

### CODE REVIEW

**Comment by tukusejssirs** [comment]
Notes:

- AFAIK, St John _De_ Ávila is not used in English; we instead you the prefered St John _of_ Ávila.
- In the [Hungarian](http://tar.liturgia.hu/Naptar/dir2019_nyomdai_withoutborder.pdf#page=173) and [Swiss/German](https://www.liturgie.ch/images/liturgie/praxis/direktorium/pdf/2020_direktorium_09_20.pdf#page=8) _Ordos_, St Hildegard of Bingen is titled an _abbess_ (in Hungarian: _apátnő_; German: _Äbtissin_), not a _virgin_, therefore I kept the _abbess_ title.
- There is a single conflict in `fr.ts` for some reason with the change from `martha_of_bethany` to `martha_of_bethany_mary_of_bethany_and_lazarus_of_bethany`; it should be resolved by removing the old key, but do check it please, @emagnier.

**Comment by tukusejssirs** [comment]
Also note that while I did my best to localise the four celebration names into all supported locales, proof the names please in your locale! Thanks in advance! :pray:

@emagnier, @Scriptura, @CodingAleCR, @jarosz, @gtortone, @HenriqueDerosa.

**Comment by gtortone** [comment]
Hi,
gregory_of_narek_abbot locale is ok for italian... but I don't find hildegard_of_bingen_abbess and john_of_avila_priest in locales/it.ts. Is it ok ?

**Comment by tukusejssirs** [comment]
> gregory_of_narek_abbot locale is ok for italian... but I don't find hildegard_of_bingen_abbess and john_of_avila_priest in locales/it.ts. Is it ok ?

Actually, I added only `gregory_of_narek_abbot` and `martha_of_bethany_mary_of_bethany_and_lazarus_of_bethany` to _all_ locales, and these two keys should be proofed.

If you wish, you can add the missing keys (either all or just a portion of them). If you want to add just those two you are talking about, you could add them to a comment to this PR and I’ll add them to `it.ts`.

Thanks anyway! :wink:

File: src/calendars/general.ts [hunk 1]
```diff
@@ -392,6 +401,15 @@ const dates = async (config: RomcalConfig): Promise<Array<LiturgicalDayInput>> =
       date: dayjs.utc(`${year}-5-3`),
       liturgicalColors: LiturgicalColors.WHITE,
     },
+    {
+      key: 'john_of_avila_priest',
+      rank: Ranks.OPT_MEMORIAL,
+      date: dayjs.utc(`${year}-5-10`),
+      liturgicalColors: LiturgicalColors.WHITE,
+      metadata: {
+        titles: [Titles.MARTYR],
```
Comment (emagnier) [inline src/calendars/general.ts]: Martyr and white? (same below in `spain.ts`)

File: src/locales/fr.ts [hunk 2]
```diff
@@ -301,7 +302,8 @@ export default {
     marie_rose_durocher_virgin:
       'Bienheureuse Marie-Rose Durocher, Religieuse, Fondatrice des Sœurs des Saints Noms de Jésus et de Marie (✝ 1849)',
     mark_evangelist: 'Saint Marc, Évangéliste',
-    martha_of_bethany: 'Sainte Marthe, Disciple du Christ (1er s.)',
+    martha_of_bethany_mary_of_bethany_and_lazarus_of_bethany:
+      'Saints Marthe, Marie et Lazare, Disciples du Christ (1er s.)',
```
Comment (emagnier) [inline src/locales/fr.ts]: 👍

**Review by emagnier — CHANGES_REQUESTED** [review]

File: src/calendars/general.ts (hunk 1 above)
Comment (tukusejssirs) [reply src/calendars/general.ts]: Oops, my mistake cause by copy-pasting without proofing. :wink:

Thanks! :pray:

**Review by tukusejssirs — COMMENTED** [review]
I’ve addressed the issue of `john_of_avila_priest`.

**Comment by tukusejssirs** [comment]
@emagnier, I’ve just noticed that I failed to base this PR on the latest commit of `_F-TypeScript`; I’ll rebase it.

**Update:** Done! :smiley:

**Comment by tukusejssirs** [comment]
@emagnier, this is also ready to merge. :smiley:

**Pull request merged**